content/en/docs/collector/building/_index.md

```diff
@@ -2,10 +2,70 @@
 title: Building custom components
 description: Instructions on how to build your own collector components
 weight: 90
+cSpell:ignore: mdatagen ottl
 ---
 
 The OpenTelemetry Collector can not only be extended by existing components, but
 also by custom components, that you develop and build on your own. Here you will
 find instructions on how to build some of those components. For additional
 details take a look into the documents contained within the
 [opentelemetry-collector-contrib repository](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/README.md).
+
+There are several
+[classes](https://github.com/open-telemetry/opentelemetry-collector/blob/67d37183e6ac9b7180fefc6dc3a55f2a75c12fba/cmd/mdatagen/main.go#L192)
+of OpenTelemetry components, such as
+
+- `receivers`: Scrapers and Listeners which
+  [ingest data](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver#readme),
+  typically used to convert data from an external source to OTLP
+- `exporters`: Ways to
+  [export data](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter#readme)
+  to non-OTLP formats, vendor-specific backends, or other Observability tools
+- `processors`: Ways to
+  [process data](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor#readme)
+  in a pipeline
+- `connectors`:
+  [Connects](https://github.com/open-telemetry/opentelemetry-collector/tree/main/connector#readme)
+  the output (exporter) of one pipeline to the input (receiver) of another in a
+  single component
+- `extensions`: Ways to
+  [augment the collector runtime](https://github.com/open-telemetry/opentelemetry-collector/blob/main/extension/README.md),
+  such as providing
```

Comment:
This duplicates https://opentelemetry.io/docs/collector/configuration/#basics, doesn't it_

Reply:
Sorry, it's still unclear to me what the audience and intent of this doc is. Is it for people building their custom components? Is it a "how to add your component to contrib"? Is it "how to contribute to the opentelemetry-collector and/or opentelemetry-collector-contrib repositories"?